instrumentation/opentelemetry-instrumentation-flask/tests/test_programmatic.py

```diff
@@ -671,6 +671,22 @@ def test_custom_request_header_added_in_server_span(self):
         self.assertEqual(span.kind, trace.SpanKind.SERVER)
         self.assertSpanHasAttributes(span, expected)
 
+    def test_repeat_custom_request_header_added_in_server_span(self):
+        headers = [
+            ("Custom-Test-Header-1", "Test Value 1"),
+            ("Custom-Test-Header-1", "Test Value 2"),
+        ]
+        resp = self.client.get("/hello/123", headers=headers)
+        self.assertEqual(200, resp.status_code)
+        span = self.memory_exporter.get_finished_spans()[0]
+        expected = {
+            "http.request.header.custom_test_header_1": (
+                "Test Value 1, Test Value 2",
```

Comment:
Is the space after the comma expected?

Reply:
yes, it's added by WSGI when it concatenates the values while setting environment variables.

Reply:
Right but I would have expected them to be comma joined by the http spec.

Looks specific to the [werkzeug test client's WSGI impl](https://github.com/pallets/werkzeug/blob/32e69512134c2f8183c6438b2b2e13fd24e9d19f/src/werkzeug/test.py#L747-L748). Just wanted to check that our instrumentation isn't doing something weird